pkg/services/provisioning/dashboards/file_reader.go

```diff
@@ -83,24 +83,24 @@ func (fr *fileReader) startWalkingDisk() error {
 	resolvedPath := fr.resolvePath(fr.Path)
 	if _, err := os.Stat(resolvedPath); err != nil {
 		if os.IsNotExist(err) {
-			return err
+			return fmt.Errorf("path %v does not exists: %v", resolvedPath, err)
```

Comment:
I am not sure if I am missing something here, but without doing this manual stack tracing in the message I have really hard time to find out what is wrong if there is an error.